ProcessLib/ComponentTransport/Tests.cmake

```diff
@@ -898,6 +898,72 @@ AddTest(
+    ConTracer_1d_pcs_1_ts_33_t_118800.000000_expected.vtu ConTracer_1d_pcs_1_ts_33_t_118800.000000.vtu Cs Cs 1e-10 1e-16
+    ConTracer_1d_pcs_1_ts_66_t_237600.000000_expected.vtu ConTracer_1d_pcs_1_ts_66_t_237600.000000.vtu Cs Cs 1e-10 1e-16
+    ConTracer_1d_pcs_1_ts_99_t_356400.000000_expected.vtu ConTracer_1d_pcs_1_ts_99_t_356400.000000.vtu Cs Cs 1e-10 1e-16
+    ConTracer_1d_pcs_1_ts_132_t_475200.000000_expected.vtu ConTracer_1d_pcs_1_ts_132_t_475200.000000.vtu Cs Cs 1e-10 1e-16
+    ConTracer_1d_pcs_1_ts_165_t_594000.000000_expected.vtu ConTracer_1d_pcs_1_ts_165_t_594000.000000.vtu Cs Cs 1e-10 1e-16
+    ConTracer_1d_pcs_1_ts_198_t_712800.000000_expected.vtu ConTracer_1d_pcs_1_ts_198_t_712800.000000.vtu Cs Cs 1e-10 1e-16
+    ConTracer_1d_pcs_1_ts_231_t_831600.000000_expected.vtu ConTracer_1d_pcs_1_ts_231_t_831600.000000.vtu Cs Cs 1e-10 1e-16
+    ConTracer_1d_pcs_1_ts_264_t_950400.000000_expected.vtu ConTracer_1d_pcs_1_ts_264_t_950400.000000.vtu Cs Cs 1e-10 1e-16
+    ConTracer_1d_pcs_1_ts_297_t_1069200.000000_expected.vtu ConTracer_1d_pcs_1_ts_297_t_1069200.000000.vtu Cs Cs 1e-10 1e-16
+    ConTracer_1d_pcs_1_ts_329_t_1184400.000000_expected.vtu ConTracer_1d_pcs_1_ts_329_t_1184400.000000.vtu Cs Cs 1e-10 1e-16
+)
+
+AddTest(
+    NAME ComponentTransport_ConTracer_2d
+    PATH Parabolic/ComponentTransport/ConTracer
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS ConTracer_2d.prj
+    WRAPPER time
+    TESTER vtkdiff
+    REQUIREMENTS NOT OGS_USE_MPI
+    RUNTIME 40
+    DIFF_DATA
+    ConTracer_2d_pcs_1_ts_30_t_108000.000000_expected.vtu ConTracer_2d_pcs_1_ts_30_t_108000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_60_t_216000.000000_expected.vtu ConTracer_2d_pcs_1_ts_60_t_216000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_90_t_324000.000000_expected.vtu ConTracer_2d_pcs_1_ts_90_t_324000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_120_t_432000.000000_expected.vtu ConTracer_2d_pcs_1_ts_120_t_432000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_150_t_540000.000000_expected.vtu ConTracer_2d_pcs_1_ts_150_t_540000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_180_t_648000.000000_expected.vtu ConTracer_2d_pcs_1_ts_180_t_648000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_210_t_756000.000000_expected.vtu ConTracer_2d_pcs_1_ts_210_t_756000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_240_t_864000.000000_expected.vtu ConTracer_2d_pcs_1_ts_240_t_864000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_270_t_972000.000000_expected.vtu ConTracer_2d_pcs_1_ts_270_t_972000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_300_t_1080000.000000_expected.vtu ConTracer_2d_pcs_1_ts_300_t_1080000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_2d_pcs_1_ts_329_t_1184400.000000_expected.vtu ConTracer_2d_pcs_1_ts_329_t_1184400.000000.vtu pressure pressure 1e-6 1e-10
```

Comment:
The same comment.

Reply:
Do you think it is too much code otherwise?